spec/index.html

```diff
@@ -2279,6 +2312,8 @@ <h3>Acknowledgments for RDF 1.2</h3>
   <h2>Changes between RDF 1.1 and RDF 1.2</h2>
 
   <ul>
+	<li>Added <a href="#section-version-announcement"class="sectionRef"></a>
+		for announcing RDF versions in RDF documents via the HTTP Content-Type header and/or syntax.</li>
```

Comment:
Please use indentation characters that are consistent with the rest of the document.
```suggestion
    <li>Added <a href="#section-version-announcement"class="sectionRef"></a>
      for announcing RDF versions in RDF documents via the HTTP Content-Type header and/or syntax.</li>
```